### PR TITLE
Adjust exsdp for offer created by client

### PR DESCRIPTION
**Pull request by Rados13**
https://github.com/membraneframework/membrane_webrtc_plugin/issues/10

### CODE REVIEW

File: lib/ex_sdp/attribute/ssrc.ex [hunk 1]
```diff
@@ -16,8 +16,10 @@ defmodule ExSDP.Attribute.SSRC do
 
   @spec parse(binary()) :: {:ok, t()} | {:error, :invalid_ssrc}
   def parse(ssrc) do
-    with [id, attribute] <- String.split(ssrc, " "),
+    with [id | [_ | _] = attribute] <- String.split(ssrc, " "),
          {:ok, id} <- Utils.parse_numeric_string(id) do
+      attribute = Enum.join(attribute, " ")
+
```
Comment (mat-hek) [inline lib/ex_sdp/attribute/ssrc.ex]:
```suggestion
    with [id, attribute] <- String.split(ssrc, " ", parts: 2),
         {:ok, id} <- Utils.parse_numeric_string(id) do
```

File: lib/ex_sdp/attribute/fmtp.ex [hunk 2]
```diff
@@ -163,6 +163,22 @@ defmodule ExSDP.Attribute.FMTP do
          do: {rest, %{fmtp | profile_id: value}}
   end
 
+  defp parse_param(["apt=" <> value | rest], fmtp), do: {rest, Map.put(fmtp, :apt, value)}
```
Comment (mat-hek) [inline lib/ex_sdp/attribute/fmtp.ex]: - add the proper field to the struct and update the type `t`
- use `%{map | key: value}` syntax
- add corresponding serializer

File: lib/ex_sdp/attribute/fmtp.ex [hunk 3]
```diff
@@ -163,6 +163,22 @@ defmodule ExSDP.Attribute.FMTP do
...
+
+  defp parse_param(["repair-window=" <> value | rest], fmtp),
```
Comment (mat-hek) [inline lib/ex_sdp/attribute/fmtp.ex:176]: as above

File: lib/ex_sdp/attribute/fmtp.ex [hunk 4]
```diff
@@ -163,6 +163,22 @@ defmodule ExSDP.Attribute.FMTP do
...
+    do: {rest, Map.put(fmtp, :repair_window, value)}
+
+  defp parse_param([head | rest], fmtp) do
+    [start_range, end_range] = String.split(head, "-")
```
Comment (mat-hek) [inline lib/ex_sdp/attribute/fmtp.ex]: move into `with`

File: lib/ex_sdp/attribute/fmtp.ex [hunk 5]
```diff
@@ -163,6 +163,22 @@ defmodule ExSDP.Attribute.FMTP do
...
+
+    with {:ok, start_range} <- Utils.parse_numeric_string(start_range),
+         {:ok, end_range} <- Utils.parse_numeric_string(end_range) do
+      {rest, Map.put(fmtp, :redundancy_range, "#{start_range}-#{end_range}")}
```
Comment (mat-hek) [inline lib/ex_sdp/attribute/fmtp.ex]: let's store that in a usable way, for example a `{start_range, end_range}` tuple

**Review by mat-hek — CHANGES_REQUESTED** [review]

File: lib/ex_sdp/attribute/fmtp.ex [hunk 6]
```diff
@@ -181,6 +204,9 @@ defimpl String.Chars, for: ExSDP.Attribute.FMTP do
         Serializer.maybe_serialize("max-br", fmtp.max_br),
         Serializer.maybe_serialize("level-asymmetry-allowed", fmtp.level_asymmetry_allowed),
         Serializer.maybe_serialize("packetization-mode", fmtp.packetization_mode),
+        Serializer.maybe_serialize("apt", fmtp.apt),
+        Serializer.maybe_serialize("repair-window", fmtp.repair_window),
+        Serializer.maybe_serialize("redundancy-range", fmtp.redundancy_range),
```
Comment (mat-hek) [inline lib/ex_sdp/attribute/fmtp.ex]: the logic for serializing the tuple to a "from-to" string seems to be missing

**Review by mat-hek — CHANGES_REQUESTED** [review]

File: lib/ex_sdp/attribute/fmtp.ex (hunk 6 above)
Comment (Rados13) [reply lib/ex_sdp/attribute/fmtp.ex:209]: After checking once again situation with redundancy-range isn't occur even with simulcast, so I will remove this param

**Review by mat-hek — APPROVED** [review]

**Pull request merged**